src/util/interpolate.ts

```diff
@@ -23,10 +24,11 @@ export function isSupportedInterpolationColorSpace(colorSpace: string): colorSpa
  * @returns interpolation fn
  * @deprecated use `interpolate[type]` instead
  */
-export const interpolateFactory = (interpolationType: 'number'|'color'|'array'|'padding'|'variableAnchorOffsetCollection') => {
+export const interpolateFactory = (interpolationType: 'number'|'color'|'projectionDefinition'|'array'|'padding'|'variableAnchorOffsetCollection') => {
```

Comment:
Can this factory be removed? It is not used in the code as far as I know...?

Reply:
i think we can remove the entire interpolate.projection() function, since it's a passthrough